lib/utils/compose_ts.ts

```diff
@@ -1015,6 +1015,9 @@ export async function makeBuildTasks(
 			infoStr = `build [${task.context}]`;
 		}
 		logger.logDebug(`    ${task.serviceName}: ${infoStr}`);
+		// Workaround for Docker v20.10 + single-arch base images. See:
+		// https://www.flowdock.com/app/rulemotion/i-cli/threads/RuSu1KiWOn62xaGy7O2sn8m8BUc
+		task.dockerPlatform = 'none';
```

Comment:
Just to clarify, looking at `resin-multibuild` I believe this means the behaviour will be to not provide the platform flag for now?

https://github.com/balena-io-modules/balena-multibuild/blob/cda88bc0f434cae3b1594ee15e532d2d455ae510/lib/resolve.ts#L48-L56

Reply:
That's right. The CLI was stuck with `resin-multibuild` v4.7.2 (could not update it) because newer versions of `resin-multibuild` provided the `platform` flag to the Docker engine, which in turn would cause `balena build` to fail with Docker v20.10 and later (that long thread ™ https://www.flowdock.com/app/rulemotion/i-cli/threads/RuSu1KiWOn62xaGy7O2sn8m8BUc ).
That issue is not resolved but I needed to update `resin-multibuild` to take advantage of the bump of `docker-progress` (Node.js v14 investigation), and generally because being unable to update `resin-multibuild` is a bad thing for `balena-cli`, as `resin-multibuild` is a fundamental component. So I came up with a way of updating `resin-multibuild` while disabling the `platform` feature. cc: @toochevere @robertgzr @dfunckt @Page-

Reply:
@klutchell, I see that I had not invited you as reviewer of the corresponding recent `resin-multibuild` PR. Check this comment: https://github.com/balena-io-modules/balena-multibuild/pull/88#discussion_r626907561